src/libraries/System.IO.FileSystem/tests/Base/SymbolicLinks/BaseSymbolicLinks.FileSystem.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public abstract class BaseSymbolicLinks_FileSystem : BaseSymbolicLinks
+    {
+        protected abstract void CreateFileSystemEntry(string path);
```

Comment:
```suggestion
        protected abstract void CreateFileSystemInfo(string path);
```
Can we change the name to above suggestion in order to avoid confusion with [`FileSystemEntry`](https://docs.microsoft.com/en-us/dotnet/api/system.io.enumeration.filesystementry?view=net-5.0)?

Reply:
No, a method with that name already exists. This method creates a `Directory` or a `File`.

Reply:
Maybe `CreateFileSystemInstance`?

Reply:
Can we name it `CreateLinkTarget`? That's the reason why you need to create a Directory or FIle, right?

Reply:
No, sometimes this method is used to create a file where the link would exist (for the tests where we try to create a link and a file already exists in that location).